main.go

```diff
@@ -7,24 +7,25 @@ import (
 	"3mdeb/RteCtrl/pkg/restServer"
 	"flag"
 	"log"
+	"strings"
 	"time"
 )
 
 var version = "0.5.2"
 
 // Flags
 var (
-	configFilePath  = flag.String("c", "/etc/RteCtrl/RteCtrl.cfg", "path to config file")
+	configFilePath = flag.String("c", "/etc/RteCtrl/RteCtrl.cfg", "path to config file")
 )
 
-func pressButton(g *gpioControl.Gpio, id int, t time.Duration) {
+func pressButton(g *gpioControl.GpioSysfs, id int, t time.Duration) {
```

Comment:
If we still use GpioSysfs interface hardcoded like this, this whole migration process seems pointless, as we still need to exposes syfss GPIO kernel interface for selected pins?